action.yml

```diff
@@ -96,6 +96,8 @@ outputs:
     description: 'Cypress Cloud URL if the run was recorded (deprecated)'
   resultsUrl:
     description: 'Cypress Cloud URL if the run was recorded'
+  testResults:
```

Comment:
Thinking about this some more, we should probably name this `runResults`.

Reply:
@mschile 

The documentation currently uses the term "test results" 
https://docs.cypress.io/guides/guides/module-api#Results

![image](https://github.com/cypress-io/github-action/assets/66998419/85d5c045-eaf0-4392-b333-8a694e1d89ba)

so if the term is changed here, it should be changed in the Module API documentation as well.